Oqtane.Client/Modules/Admin/Jobs/Add.razor

```diff
@@ -18,7 +18,12 @@
             <Label For="type" HelpText="Enter the job type" ResourceKey="Type">Type: </Label>
         </td>
         <td>
-            <input id="type" class="form-control" @bind="@_jobType" />
+            <select id="type" class="form-control" @bind="@_jobType">
+                @foreach (var item in _jobTypeList)
+                {
+                    <option value=@item>@item</option>
```

Comment:
Surround the `@item` with double quotes

Reply:
You are right, I will fix it.

Reply:
Fixed.

Reply:
Need to be fixed

Reply:
no matter if double quotes is here or not, resulting html is same. Blazor renders attribute correctly.
![image](https://user-images.githubusercontent.com/7838468/104899227-df2bbd80-597a-11eb-9b7a-c5a148df0963.png)